.github/workflows/endorlabs.yml

```
@@ -13,6 +13,7 @@ on:
     - cron: '26 20 * * 6'
 jobs:
   scan:
+    if: ${{ secrets.ENDOR_NAMESPACE != '' }}
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)